WalletWasabi.Backend/SendPushService.cs

```diff
@@ -80,7 +81,7 @@ public async Task SendNotificationsAsync(bool isDebug)
 			var tokens = context.Tokens
 				.Where(t => t.IsDebug == isDebug)
 				.Distinct();
-			foreach (var token in tokens)
+			await Task.WhenAll(tokens.Select(async token =>
```

Comment:
Parallelizing the calls

Reply:
unfortunately the `async token => ...` lambda cannot be converted to an expression tree as argument to `tokens.Select(`